src/pages/Organizations/organizationInventory.js

```diff
@@ -10,30 +10,60 @@ import getOrganizationById from "../../queries/Organization/getOrganizationById.
 import getEconomicResourceById from "../../queries/EconomicResource/getEconomicResourceById.tsx";
 import {isNullOrUndefined} from "util"
 import {Item, Button, Loader} from 'semantic-ui-react'
-
+import {connect} from 'react-redux';
+import getMyAgent from "../../queries/Agent/getMyAgent"
 let default_image = require("../../resources/default_resource_img.jpg");
+let orgId = -1;
+let createItem;
+
+const AddItemButton = getMyAgent(({agent, loading, error}) => {
+    if (loading) {
```

Comment:
The agent data should be saved in redux so we don't have to continually waste network traffic, but it's not a big enough issue yet  to put the merge on hold.